hugegraph-core/src/main/java/org/apache/hugegraph/util/CompressUtil.java

```diff
@@ -57,6 +57,7 @@
 public final class CompressUtil {
 
     /**
+     * TODO: ensure the refer is valid & clear
```

Comment:
We looked at the code on the blog, maybe we can remove the reference, because the code is a common util